test/support/linearizability.ex

```diff
@@ -28,30 +28,24 @@ defmodule Craft.Linearizability do
     ]
   end
 
-  defmodule TestModel do
```

Comment:
i was planning on extracting the linearizability tester into its own repo, since it's not specific to raft. the TestModel here is the behaviour for that (it excludes the log_index since that's raft-specific).